lib/workers/pr/index.ts

```diff
@@ -90,6 +90,36 @@ export async function addAssigneesReviewers(
   }
 }
 
+async function getExistingPr(
+  branchName: string,
+  prTitle: string
+): Promise<Pr | null> {
+  const existingPr = await platform.getBranchPr(branchName);
+  if (existingPr) {
+    logger.debug('Found existing PR');
+    return existingPr;
+  }
+
+  if (platform.reopenPr) {
+    const closedPr = await platform.findPr({
+      branchName,
+      prTitle: `${prTitle} - autoclosed`,
+      state: 'closed',
+    });
+    if (closedPr && closedPr.number) {
```

Comment:
Need to check for merged state, as it can't be reopened
```suggestion
    if (closedPr?.number) {
```